groups/edpr-iit.yml

```
@@ -19,6 +19,7 @@ edpr-iit/purchases:
   - "simbamford"
   - "marco-monforte"
   - "arrenglover"
+   - "ejanotte"
```

```suggestion
  - "ejanotte"
```

We need to comply with the indentation.